src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -820,7 +820,9 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
 
     ASR::symbol_t* resolve_symbol(const Location &loc, const std::string &sub_name) {
         SymbolTable *scope = current_scope;
-        ASR::symbol_t *sub = scope->resolve_symbol(sub_name);
+        std::string lower_sub_name = sub_name;
+        std::transform(lower_sub_name.begin(), lower_sub_name.end(), lower_sub_name.begin(), [](unsigned char c) { return std::tolower(c); });
+        ASR::symbol_t *sub = scope->resolve_symbol(lower_sub_name.c_str());
```

Comment:
```suggestion
       ASR::symbol_t *sub = scope->resolve_symbol(to_lower(sub_name));
```
Something like this should do the trick

Reply:
Oh right. I tried this but was getting some error. It's working fine now. Thanks!